assets/src/edit-story/app/story/actions/useAddBlankPage.js

```diff
@@ -15,7 +15,7 @@ function useAddBlankPage( {
 	clearSelection,
 } ) {
 	const addBlankPage = useCallback( () => {
-		const newPage = createPage( { index: pages.length } );
+		const newPage = createPage();
```

Comment:
Removed adding `index` as a param since otherwise we'd have to update all the index params once the order changes or a Page is deleted. Getting the actual index from the actual `pages` seems to make more sense at this moment.

Reply:
@barklund Do you see any concerns with removing the `index` param from here in favor of using index from the actual updated list of pages?